Client/Frontend/Intro/IntroViewModel.swift

```diff
@@ -44,37 +44,51 @@ struct IntroViewModel: OnboardingViewModelProtocol, FeatureFlaggable {
         }
     }
 
-    var enabledCards: [InformationCards] {
-        guard let wallpaperVersion: WallpaperVersion = featureFlags.getCustomState(for: .wallpaperVersion) else {
-            return [.welcome, .wallpapers, .signSync]
-        }
+    var isMROnboardingVersion: Bool {
```

Comment:
Can we rename this to be more specific? And additionally create a task to remove this once the MR is released? Don't want to end up at the next MR and still have this in place and not knowing with MR it was for.

Reply:
Update name and I've updated the acceptance criteria of a future task to remove this logic